jenkins/bootstrap.py

```diff
@@ -385,10 +385,17 @@ def metadata(repo, artifacts):
     if not meta or not isinstance(meta, dict):
         meta = {}
     meta['repo'] = repo
+
+    try:
+        commit = call(['git', 'rev-parse', '--short=14', 'HEAD^{commit}'], output=True)
```

Comment:
How is "HEAD^{commit}" different from HEAD? We might as well use the full commit hash here (no --short)-- you can truncate it for testgrid display.